src/app/routes/article/getInitialData/index.ts

```diff
@@ -51,33 +49,6 @@ export default async ({
       ...(!isLocal && { agent, optHeaders }),
     });
 
-    // Ensure all local CPS fixture and test data is in the correct format
-    if (isLocal && pageType === 'cpsAsset') {
-      const secondaryData = await getAdditionalPageData({
-        pageData: json,
-        service,
-        variant,
-        env,
-      });
-
-      json = {
-        data: {
-          article: json,
-          // Checks for data mocked in tests, or data from fixture data
-          secondaryData: json?.secondaryData ?? {
-            topStories: secondaryData?.secondaryColumn?.topStories,
-            features: secondaryData?.secondaryColumn?.features,
-            mostRead: secondaryData?.mostRead,
-            mostWatched: secondaryData?.mostWatched,
-          },
-        },
-      };
-    }
-
-    if (!json?.data?.article) {
-      throw handleError('Article data is malformed', 500);
```

Comment:
this statement isn't in the scope of the `isLocal` if statement, should it have been removed?

Reply:
That's fair, I'll add it back in, as it will hopefully catch problems with fixture data not being correct